### ISSUE TITLE
Alpha Testing: Feature Request--Retrace a Wrong Move (Gameplay Mechanics)

### ISSUE THREAD

**Issue by Deepaklather** [jaimeenahn/goHomeBulls#140]
Can there be a feature to retrace a wrong move? (i.e. if we a push a drunken bull in the wrong direction, then there should be an option/mechanic to retrace that particular move, basically bringing that bull to the original coordinates. I assume this could be accomplished through a point/item/credit system, where a player loses a point (-1) for retracing a move!

**Comment by zhaosec** [comment]
Yeah, we have never come up with this idea. We will consider adding this feature in the future.

**Comment by ubbh** [comment]
That is a good idea to make the game more interesting. Thank you for the suggestion, we will arrange it as soon as possible.
